docs: fix typos (#84)
cilium/hubble-otel: USER_GUIDE_KIND.md

to run as sidecars of the demo app.

The demo app used is [podinfo](https://github.com/stefanprodan/podinfo), it's instrumented using

> OpenTelemetry SDK. There is a frontend service as well as backend which are going to be configured

Reviewer: ```suggestion
OpenTelemetry SDK. There is a frontend and backend configured
```

Small suggestion to eliminate some wordiness
